simplified-app-shared/src/main/java/org/nypl/simplified/app/LoginDialog.java

```diff
@@ -405,6 +418,15 @@ public void onAccountLoginFailureDeviceActivationError(final String message)
       final Account account = new AccountsRegistry(getActivity()).getAccount(Integer.valueOf(account_id));
       books = Simplified.getBooks(account, getActivity(), Simplified.getCatalogAppServices().getAdobeDRMExecutor());
       docs = Simplified.getDocumentStore(account, getActivity().getResources());
+
+      if (account.supportsBarcodeScanner()) {
```

Comment:
One thing I'm wondering: if these two key/values are missing in Accounts.json, would trying to access this variable from `account` throw a null exception? I haven't tried and don't know enough about Java to know if it falls back gracefully.

If it does crash, just for these two variables, we should provide defaults of `false` for scan and display in `Accounts.java`. The whole class is getting refactored soon with the addition of the account registry.

Since outside contributors haven't been accustomed to adding these properties for other libraries, it's a likely point of failure.

Reply:
If the pair is missing, it is defaulting to false on these account methods.